src/data/defaults/progress/assaultRifles.js

```diff
@@ -54,4 +54,18 @@ export default {
 		'Savannah': false,
 		'Petri Digital': false,
 	},
+
+	'M13B': {
+		'TBA': false,
+		'TBA2': false,
+		'TBA3': false,
+		'TBA4': false,
+	},
+
+	'Chimera': {
+		'TBA': false,
+		'TBA2': false,
+		'TBA3': false,
+		'TBA4': false,
+	},
```

Comment:
Leave empty for now.
```suggestion
	'M13B': {},

	'Chimera': {},
```